pkg/prowgen/prowgen_konflux.go

```diff
@@ -295,7 +288,6 @@ func GenerateKonfluxServerlessOperator(ctx context.Context, openshiftRelease Rep
 			},
 			Excludes:       b.Konflux.Excludes,
 			ExcludesImages: b.Konflux.ExcludesImages,
-			FBCImages:      b.Konflux.FBCImages,
```

Comment:
I would leave it here

Reply:
in https://github.com/openshift-knative/hack/pull/341, it looks like we need it

Reply:
I removed it only from the [`prowgen.Konflux`](https://github.com/openshift-knative/hack/blob/295c5f17bcbc79d3d0035737f6f4abb467bc70b8/pkg/prowgen/prowgen_config.go#L81) struct, but kept it in the [`konfluxgen.Config`](https://github.com/openshift-knative/hack/blob/295c5f17bcbc79d3d0035737f6f4abb467bc70b8/pkg/konfluxgen/konfluxgen.go#L59) struct.